dev/io.openliberty.microprofile.health.internal.common/resources/io/openliberty/microprofile/health/resources/Health.nlsprops

```diff
@@ -79,6 +79,18 @@ file.healthcheck.file.delete.fail.CWMMH0106W=CWMMH0106W: Unable to delete existi
 file.healthcheck.file.delete.fail.CWMMH0106W.explanation=The file-based health checks require the creation and use of the files "live", "ready" and "started" in the /health directory. For the file-based health checks to function correctly, these files must not exist when the runtime starts. The runtime has attempted to delete the existing file was unable to.
 file.healthcheck.file.delete.fail.CWMMH0106W.useraction=The user must identify the root cause preventing the runtime from deleting the files.
 
+file.healthcheck.system.inconsistency.CWMMH0107W=CWMMH0107W: The runtime has detected inconsistencies with the local file-based health check mechanism. The file-based health check function will be disabled.
```

Comment:
Change to:

file.healthcheck.system.inconsistency.CWMMH0107W=CWMMH0107W: The runtime detected inconsistencies with the local file-based health check mechanism. The file-based health check function is disabled.
file.healthcheck.system.inconsistency.CWMMH0107W.explanation=Inconsistencies were detected with the file system. For example, the "started" file is present before the runtime creates the "started" file.
file.healthcheck.system.inconsistency.CWMMH0107W.useraction=Delete the <server.output.dir>/health directory and restart the server.